src/ui/server/controllers/insurance/business/company-details/index.ts

```diff
@@ -160,7 +160,7 @@ const post = async (req: Request, res: Response) => {
      * - Change answers form POST.
      * - Check/change answers form POST.
      */
-    const tradingAddressIsRequired = sanitisedHasTradingAddress && !application.company.differentTradingAddress.fullAddress;
+    const tradingAddressIsRequired = sanitisedHasTradingAddress;
```

Comment:
I don't think this is correct - recently fixed a bug where the POST would redirect to "trading address" if full address already exists: https://ukef-dtfs.atlassian.net/browse/EMS-2557

?

Reply:
discussed.  Additional rows have been removed so must always go to this page if differentTradingAddress yes is selected